docs/Makefile

```diff
@@ -25,19 +25,20 @@ clean:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@echo "Cleaning build folder..."
-	rm -rf "$(BUILDDIR)"
+	# No longer removing the build each time.
+	# Sphinx natively supports caching through a .doctrees dir in the build dir
+
 	mkdir -p "$(DOCSBUILDDIR)"
 
 	@echo "Copying docs & config to build folder..."
-	cp -a "$(DOCSDIR)" "$(BUILDDIR)"
+	# cp -a "$(DOCSDIR)" "$(BUILDDIR)"
```

Comment:
this should either be added back or removed

Reply:
Removed it